torchsde/_core/adjoint.py

```diff
@@ -195,27 +90,22 @@ def backward(ctx, *grad_outputs):
                 dt_min=dt_min,
                 options=adjoint_options
             )
+            aug_state = misc.flat_to_shape(aug_state[1], shapes)  # Unpack the state at time -ts[i - 1].
+            aug_state[0] = ys[i - 1]
+            aug_state[1] += grad_ys[i - 1]
```

Comment:
In principle is `sdeint_adjoint` going to support double backward? I've been thinking that this may actually be a very important thing for us to support wrt our current project.
(I hadn't mentioned it as I didn't want to add to your workload. But here everything looks like it should pretty much work. At least in principle; not sure how accurate the double-gradients will actually be.)

I'd suggest changing this to an out-of-place update just in case, and changing the call to `sdeint.integrate` to `_SdeintAdjointMethod.apply`, and I think we should be good to go.

Reply:
I can change to out-of-place update for now. Though, I'm not inclined to switch from `sdeint.integrate` to `_SdeintAdjointMethod.apply` for now. 

I like testing :), and I think we should do that in another PR after testing that nothing breaks just to be safe.

Reply:
Fair enough.